src/event.js

```diff
@@ -103,3 +105,51 @@ module.exports.addLabelToMetadata = function addLabelToMetadata(event, key, valu
         event.getResource().getMetadataMap().set('labels', labelsJson);
     }
 };
+
+/**
+ * Create and initialize a new serverless event in the epsagon format.
+ * @param {string} resourceType resourceType name
+ * @param {string} name Event name
+ * @param {string} operation Operation name
+ * @param {string} origin Origin name (optional)
+ * @returns {Object} Object with serverlessEvent and event started time.
+ */
+module.exports.initializeEvent = function initializeEvent(resourceType, name, operation, origin) {
+    const startTime = Date.now();
+    const resource = new serverlessEvent.Resource([
+        name,
+        resourceType,
+        operation,
+    ]);
+    const slsEvent = new serverlessEvent.Event([
+        `${resourceType}-${uuid4()}`,
+        utils.createTimestampFromTime(startTime),
+        null,
+        origin || resourceType,
```

Comment:
why?

Reply:
line 128 - it's for mark the event as a runner/trigger in case of origin exists.